types/scene/wlr_scene.c

```diff
@@ -1673,6 +1880,8 @@ static bool scene_node_invisible(struct wlr_scene_node *node) {
 		struct wlr_scene_shadow *shadow = wlr_scene_shadow_from_node(node);
 
 		return shadow->color[3] == 0.f;
+	} else if (node->type == WLR_SCENE_NODE_OPTIMIZED_BLUR) {
+		return false;
```

Comment:
The node should always be "visible" for us to render the blur onto the texture.

In the "workspace_get_blur" function we check if there even are any nodes that want optimized blur. If not, we won't render it.

This is very janky though with it being a node and such so an alternative to this would be nice xD

We could maybe create add a field in the root called "optimized_blur_tree" (SwayFX would have the bottom layer tree) or something and just check each render itr if that whole tree has rendered and render the optimized blur?

Not sure how we would check which tree is currently rendering though